WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingHomeAdapter.kt

```diff
@@ -18,6 +20,8 @@ import javax.inject.Inject
 private const val headerViewType: Int = 1
 private const val homeItemViewType: Int = 2
 private const val submitButtonViewType: Int = 3
+private const val storyTitleViewType: Int = 4
```

Comment:
Nitpick: I wonder why we don't use enums and use them as Ints so it's easier to derive a `viewType` for the viewHolder (and the code looks cleaner and there wouldn't be ambiguity / overlap chance). Feel free to leave as is if that's the way things are being handled for other viewtypes in other places though, but thought I'd comment on my thoughts.

Reply:
Thanks! This is a good idea that's worth exploring! I didn't change this due to it being the approach being used so I will keep this in mind for refactoring 🙏